subcommand/mesh-init/command_test.go

```diff
@@ -148,6 +186,22 @@ func TestRun(t *testing.T) {
 			require.NoError(t, err)
 			require.True(t, cmp.Equal(expectedProxyServiceRegistration, proxyService, agentServiceIgnoreFields))
 
+			if c.checks != nil {
+				// Validate the check is present
+				checks, err := consulClient.Agent().Checks()
+				require.NoError(t, err)
+				for _, expCheck := range c.checks {
+					require.Contains(t, checks, expCheck.CheckID)
```

Comment:
Is there a way to assert against the check definition to make sure the fields got set?